src/gwt/src/org/rstudio/core/client/widget/ScrollableToolbarPopupMenu.java

```diff
@@ -59,7 +63,7 @@ public void ensureSelectedIsVisible()
    
    public int getSelectedIndex()
    {
-      return menuBar_.getSelectedIndex();
+      return ((ScrollableToolbarMenuBar)menuBar_).getSelectedIndex();
```

Comment:
Rather than directly accessing the superclass `menuBar_` member, and casting it, consider making it private in `ThemedPopupPanel`, and have a protected method that returns it. Then you can override by return-type in the subclasses and have those return the appropriate type with no need for any casts.